### ISSUE TITLE
[Bug]: Creating a Shell via POST:/shells is not persistent

### ISSUE THREAD

**Issue by BohnSeb** [eclipse-aaspe/aasx-server#240]
### Describe the bug

Creating a Shell over the REST API creates the shell successfully. However after a server restart the shell is no longer existing. I believe this was not the case in the v2023-09-13A release. Using DB instead doesn't help, it doesn't get saved.

### To reproduce

Create a Shell using a POST request and restart the server. The shell will no longer be displayed or available via rest requests.

### Screenshots

```bash
![DESCRIPTION](LINK.png)
```

### Expected Behaviour

The Shell should be saved on file system in the data-path specified folder or in the database, so it is available after restarting the server.

### Version

2023-11-17A

### Operating System

Windows

**Comment by BohnSeb** [comment]
There is a workaround for this. If you download the package one time it get's created locally. However if things get manipulated via the rest api the changes are then not written to the harddrive, even if you apply the same trick again

**Comment by andercrzz** [comment]
Hi  @BohnSeb, have you solve it? I need this feature for my project and i think the bug is still unresolved. Thanks

**Comment by BohnSeb** [comment]
> Hi  @BohnSeb, have you solve it? I need this feature for my project and i think the bug is still unresolved. Thanks

No, I only found the workaround. I cannot contribute a bugfix as I am not familiar with the programming language
